### ISSUE TITLE
Use stroke black to better accessibility / contrast

### ISSUE THREAD

**Issue by devslayertech** [reviewx/Contribute#29]
![image](https://user-images.githubusercontent.com/100967476/163449220-f057b31b-8e2e-4d69-a398-c8e3f8912ed2.png)

```
.rx-review-form-area-style-1 .rx_star_rating .icon-star,
.rx-review-form-area-style-1 .rx_star_rating:not(:checked) > label:hover .icon-star, 
.rx_star_rating:not(:checked) > label:hover ~ label .icon-star {
    stroke: black !important;
}
```

**Comment by amin-abir** [comment]
Thanks for your suggestion.
